src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -72,11 +72,11 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
                 continue;
             }
 
-            String pathAdjusted = file.absolutePath().replace('\\', '/');
+            String pathAdjusted = file.absolutePath().replace('\\', '/').toLowerCase();
```

Comment:
@Pablissimo 
It's fix problem with ``C:/`` SonarQube return ``C:/`` but TSLint return ``c:/``